src/main/java/com/github/princesslana/eriscasper/gateway/commands/UpdatePresence.java

```diff
@@ -0,0 +1,27 @@
+package com.github.princesslana.eriscasper.gateway.commands;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.princesslana.eriscasper.data.resource.Activity;
+import io.reactivex.annotations.Nullable;
+import org.immutables.value.Value;
+
+/**
+ * @see <a href="https://discordapp.com/developers/docs/topics/gateway#update-status">
+ *     https://discordapp.com/developers/docs/topics/gateway#update-status</a>
+ */
+@Value.Immutable
+public interface UpdatePresence {
+  @JsonProperty("since")
+  @Nullable
```

Comment:
Why `@Nullable` instead of `Optional`?

Reply:
Passing through null afaik means something different than not passing it through at all.